utils/resize.py

```diff
@@ -94,17 +103,36 @@ def parse_args():
     for img in dat[u'_via_img_metadata']:
         regions = dat['_via_img_metadata'][img]['regions']
         print(regions)
-        for r in regions:
-            x_original = [x for x in r['shape_attributes']['all_points_x']]
-            y_original = [y for y in r['shape_attributes']['all_points_y']]
-            x_scaled = [x * new_width/original_width for x in x_original]
-            y_scaled = [y * new_height/original_height for y in y_original]
-            print(r['shape_attributes']['all_points_y'])
-            print(y_original)
-            print(y_scaled)
-            r['shape_attributes']['all_points_y'] = y_scaled
-	    r['shape_attributes']['all_points_x'] = x_scaled
-            print(r['shape_attributes']['all_points_y'])
+        polygon_flag='False'
```

Comment:
```suggestion
        polygon_flag = False
```

Similarly, 
```
        polygon_flag = True
```

when setting `True`. Then, 
```
        if polygon_flag:
```
instead of `if polygon == True`.

In case you compare with False, it will be 
```
        if not polygon_flag:
```
instead of `if polygon == False`.